### PR TITLE
added lambda runtime table

### PR DESCRIPTION
**Pull request by zagronitay**

### CODE REVIEW

File: resources/lambda_functions.go [hunk 1]
```diff
@@ -916,6 +916,19 @@ func LambdaFunctions() *schema.Table {
 					},
 				},
 			},
+			{
+				Name:        "aws_lambda_runtimes",
```
Comment (roneli) [inline resources/lambda_functions.go:920]: why not just embed as a text array? also what is the relational FK here? can each function have only 1 runtime? if so just save it has a text field as runtime? Basically you will call this resolve each time per function and it will all conflict each time.

**Review by roneli — CHANGES_REQUESTED** [review]

**Review by roneli — APPROVED** [review]
LGTM

**Comment by roneli** [comment]
Would also add a test to this so we have sanity it works, no need to mock, just call and see it not empty
